docs/config.md

```diff
@@ -126,6 +126,7 @@ due to lack of stable semantic convention.
 
 | ID                    | Instrumented library                                                                                                                                                                            | Supported versions | Instrumentation type    | Status                                                                                                                            |
 |-----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------------|-------------------------|-----------------------------------------------------------------------------------------------------------------------------------|
+| `AZURE`               | Azure SDK                                                                                                                                                                                       | *                  | source                  | [Experimental](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/versioning-and-stability.md) |
```

Comment:
Consider changing `AZURE` key to `AZURESDK`
I suppose that in the future we could have more Azure related instrumentations.

Reply:
I didn't use `AZURESDK` because there were also `MONGODB` and `NPGSQL`. `SDK` seems like a redundant suffix because... of course it's a SDK.

I don't have a strong insist about it though.